roles/postgres/defaults/main.yml

```diff
@@ -1,5 +1,4 @@
 ---
-postgres_version: 13
 _postgres_image: postgres
```

Comment:
Don't we have to change the postgresql container image value here too ?

Reply:
The `postgres_version` variable was replaced by `supported_pg_version` variable for consistency across the other operators.

Reply:
I'm talking about the other variable : `_postgres_image` ;)

Reply:
ahh, woops. Fixed now, thanks!